includes/functions.php

```diff
@@ -30,16 +30,22 @@ function get_directory_blog_id () {
 
 if (!function_exists('get_site_directory_terms')) :
     /**
-     * Gets all categories in the site directory.
+     * Gets categories in the site directory.
+     *
+     * @param array $slugs Optional. Slugs of categories to retrieve. Default is to retrieve all categories.
+     * @param array $exclude Optional. IDs of categories to exclude.
      *
      * @uses get_terms()
      *
      * @return array|false|WP_Error
      */
-    function get_site_directory_terms () {
+    function get_site_directory_terms ($slugs = array(), $exclude = array() ) {
```

Comment:
Ok. I added a `wp_parse_args()` to merge the default parameters and those passed to the function.